modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/test/js/components/LayoutProvider/handlers/fieldDeletedHandler.es.js

```diff
@@ -68,14 +72,14 @@ describe('LayoutProvider/handlers/fieldDeletedHandler', () => {
 
 			const clearAllConditionFieldValuesSpy = jest.spyOn(
 				RulesSupport,
-				'clearAllConditionFieldValues'
+				'formatRules'
 			);
 
 			clearAllConditionFieldValuesSpy.mockImplementation(() => []);
 
-			fieldDeletedHandler.handleFieldDeleted(state, event);
+			fieldDeletedHandler.handleFieldDeleted({}, state, event);
 
-			expect(clearAllConditionFieldValuesSpy).toHaveBeenCalled();
+			expect(clearAllConditionFieldValuesSpy).not.toHaveBeenCalled();
```

Comment:
@alinedoleron, same mistake from the previous pull, the variable spying the method was named inconsistently. Instead of `clearAllConditionFieldValuesSpy` it should be `formatRulesSpy`.